feature-libs/cart/import-export/components/export-entries/export-entries.service.ts

```diff
@@ -108,4 +129,15 @@ export class ExportEntriesService {
       })
     );
   }
+
+  downloadCsv(entries: string[][]) {
+    if (this.exportConfig.messageEnabled) this.displayExportMessage();
+
+    this.exportCsvService.downloadCsv(
+      this.exportCsvService.dataToCsv(entries),
+      this.exportConfig.fileName,
```

Comment:
should be not a current saved cart name or some configured compatible (the same way as for import, e.g date). Not sure about active cart

Reply:
Don't want to generate file names for export (or at least in this PR). I would rather let customer to decide how he wants to name them (I replaced `data` with `cart`) which fit for both cases I guess.